test/jdk/TEST.groups

```diff
@@ -381,9 +382,11 @@ jdk_svc = \
 
 jdk_foreign = \
     java/foreign \
-    jdk/internal/reflect/CallerSensitive/CheckCSMs.java \
-    -java/foreign/TestMatrix.java \
-    -java/foreign/TestUpcallStress.java
+    jdk/internal/reflect/CallerSensitive/CheckCSMs.java
```

Comment:
Please exclude the `jdk_foreign_stress` group from the regular `jdk_foreign` test group as well. We regularly run `jdk_foreign` as part of the normal dev cycle.